crates/sui-graphql-rpc/src/types/query.rs

```diff
@@ -79,6 +81,7 @@ impl Query {
             .extend()
     }
 
+    /// The address objects corresponding to this address.
```

Comment:
This is another type where I'm not sure if it needs a doc comment. In any case, I think it should be a singular object, but also not sure about "address object corresponding to this address", since an address could be the id of an object, or just an address/ account

Reply:
> Look-up an Account by its `SuiAddress`.